### PR TITLE
CREATE MILESTONE

### PR DESCRIPTION
**Pull request by nashvegasnate**
This PR represents the Milestone of CREATE which include form and input functionality with the following categories. Each category appears and updated in Firebase which dynamically manipulates the DOM.

CREATE Planes: An 'Add Planes' button is clicked a form will show on the the form section asking to input an image, name, capacity and model number to create a card. When submitted, a card with the submitted card corresponding information will be dynamically added to the DOM. 

CREATE Airports: When the 'Add Airports' button is clicked a form will show on the the form section asking to input an image, name and airport location to create a card. When submitted, a card with the submitted card and corresponding information will be dynamically added to the DOM. 

CREATE Crews: When the 'Click To Apply' button is clicked a form will show on the the form section asking to input an image, crew title, name and airport location to create a card. When submitted, a card with the submitted card and corresponding information will be dynamically added to the DOM.

CREATE Food Service: When the 'Add A Meal' button is clicked a form will show on the the form section asking to input an image, name of food name and price of food to create a card. When submitted, a card with the submitted card and corresponding information will be dynamically added to the DOM.

CREATE Baggage: When the 'Check-In Bag' button is clicked a form will show on the the form section asking to input an image, baggage number and weight of bag to create a card. When submitted, a card with the submitted card and corresponding information will be dynamically added to the DOM.

##Link to deployed APP: 
https://nss-pan-am-2021.netlify.app/

##Why is this change required?
Required to achieve MVP on the Pan-Am Experience assignment due on Mon/Tue

##Related Issues
#63 
#62 
#61 
#59 
#56

##How can this be tested? 
Visit the deployed link, login and click through the Navbar and follow the user story description above to explore each categories listed.

##Screenshot
V2 Create

## Types of changes
- [ ] Bug fix
- [x] New feature
- [ ] Feature Breaking change

### CODE REVIEW

File: src/javascripts/components/forms/addFoodForm.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const addFoodForm = () => {
+  // document.querySelector('#modal').innerHTML = '';
```
Comment (TrinityChristiana) [inline src/javascripts/components/forms/addFoodForm.js]: If this code isn't going to be used in the future, let's get rid of it.

File: src/javascripts/events/foodDomEvents.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      addFoodForm();
+    }
+    // CLICK EVENT FOR SUBMITTING FORM FOR ADDING FOOD
+    if (e.target.id.includes('submit-food')) {
+      e.preventDefault();
+      const foodObject = {
+        foodImage: document.querySelector('#foodImage').value,
+        foodName: document.querySelector('#foodName').value,
+        foodPrice: document.querySelector('#foodPrice').value,
+        // uid: firebase.auth().currentUser.uid
```
Comment (TrinityChristiana) [inline src/javascripts/events/foodDomEvents.js]: If this code isn't going to get used, let's get rid of it.

File: src/javascripts/events/foodDomEvents.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      };
+      console.warn(foodObject);
```
Comment (TrinityChristiana) [inline src/javascripts/events/foodDomEvents.js]: There's still a console.warn in here.

File: src/styles/airport.scss [hunk 4]
```diff
@@ -1,22 +1,22 @@
-@import "~bootstrap/scss/bootstrap";
-@import "~@fortawesome/fontawesome-free/css/all.min.css";
+// @import "~bootstrap/scss/bootstrap";
```
Comment (TrinityChristiana) [inline src/styles/airport.scss]: If this code isn't going to be used in the morning, let's get rid of it.

**Review by TrinityChristiana — COMMENTED** [review]
Everything looks great and everything's functional. These are just a few things that need to be removed before I can approve this PR!

**Comment by danicrosby** [comment]
@TrinityChristiana made required tweaks! 

https://nss-pan-am-2021.netlify.app/

**Review by TrinityChristiana — APPROVED** [review]
😎

**Pull request merged**